proximo/proximo_source.go

```diff
@@ -134,31 +135,27 @@ func (ams *asyncMessageSource) ConsumeMessages(ctx context.Context, messages cha
 		for {
 			in, err := stream.Recv()
 			if err != nil {
-				if err != io.EOF && grpc.Code(err) != codes.Canceled {
-					return err
+				if err == io.EOF || status.Code(err) == codes.Canceled {
```

Comment:
i read something in the [grpc release notes](https://github.com/grpc/grpc-go/releases/tag/v1.19.0) about io.EOF no longer surfacing in the client today. maybe worth checking?

Reply:
not sure it applies here, maybe something for another PR